scripts/get_trello_members.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+"""
+Get all members of all boards owned by organisations you are part of.
+"""
+import os
+import requests
+import json
+
+BASE_URL = "https://api.trello.com/1"
+HEADERS = {"Accept": "application/json"}
+QUERY = {
+    "key": os.environ["TRELLO_KEY"],
+    "token": os.environ["TRELLO_TOKEN"],
+}
+
+
+def get_organisations():
+    """
+    All the DMp Trello organisations.
+    """
+    return {
+        "DMP 2": "5e70a808f51fba20f624d9e2",  # pragma: allowlist secret
+        "DMP 3": "5eb0095a4b182882dbb46652",  # pragma: allowlist secret
+        "DMP 4": "5f11bf47fb7c2f529c42193a",  # pragma: allowlist secret
+        "Digital Marketplace - up to May 2019": "56ab94d6239d393aafcc0dd7",  # pragma: allowlist secret
+        "Digital Marketplace Missions": "5cd16b6993f4f0616b48c101",  # pragma: allowlist secret
```

Comment:
Some additional orgs:

```suggestion
        "Digital Marketplace Missions": "5cd16b6993f4f0616b48c101",  # pragma: allowlist secret
        "Digital Marketplace UR": "5cc18c22ae0b160c5229e07e",  # pragma: allowlist secret
        "DMP EMPATHY LAB": "5e99bd017496273606e74187",  # pragma: allowlist secret
```